src/components/desktop/desktop.vue

```diff
@@ -16,15 +25,20 @@
             </LocationProvider>
           </div>
           <LocationProvider location="predictive_layer">
-            <RelatedTags v-if="!$x.isEmpathizeOpen && $x.relatedTags.length > 0" />
+            <RelatedTags v-if="$x.relatedTags.length > 0" />
```

Comment:
The 'if' is removed because we were hidden them when we open the empathize, so there were an animation increasing and decreasing the header, then to avoid that, we leave there the related tags always